statics/js/components/capture-form.js

```diff
@@ -68,9 +68,15 @@ Vue.component('capture-form', {
               <label for="port">Port</label>\
               <input id="port" type="number" class="form-control input-sm" v-model.number="port" min="0"/>\
             </div>\
+            <div class="form-group" v-if="captureType == \'ovssflow\'">\
+              <label for="samplingrate">Flow Sampling Rate</label>\
+              <input id="samplingrate" type="number" class="form-control input-sm" v-model.number="samplingrate" min="1"/>\
```

Comment:
you can set the max value also similar to the validator.

Reply:
+1

Reply:
Thank You!